ls1intum/Artemis
# Bugfix/gitlab change permission for all programming exericses

### Checklist
#### General


#### Server


### Motivation and Context


Quick fix for #4081

### Description

Tutors are now added to exam exercises in GitLab as well

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Tutor

1. Create an exam with a programming exercise in an environment that uses GitLab
2. Add a new user as tutor in the course
3. Check that the new tutor can access the repositories of the exercise in GitLab

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage
No changes
